global_vars/noninteractive/amazon-site.yml

```diff
@@ -0,0 +1,68 @@
+---
+# Example site specific configuration for a noninteractive AWS deployment.
+#
+# Copy this and edit it as needed before running streisand-new-cloud-server.
+#
+
+streisand_noninteractive: true
+confirmation: true
+
+# The SSH private key that Ansible will use to connect to the Streisand node.
+#
+# The corresponding public key must be added to the Digital Ocean control panel
```

Comment:
This paragraph refers to Digital Ocean but it's an Amazon config example.

Reply:
The other defaults files (minus the actual Digital Ocean defaults file) should have this comment fixed as well.